Sources/LeafKit/Data/TemplateData.swift

```diff
@@ -2,9 +2,22 @@ import Foundation
 
 /// TemplateKit's supported serializable data types.
```

Comment:
Aside from comments below on this file, all changes are whitespace/switch formatting cleanup for readability